src/main/resources/scripts/teleport with compass.sk

```diff
@@ -3,14 +3,8 @@
 # with a compass like in WorldEdit.
 #
 
-aliases:
-	# blocks without collision as of Minecraft 1.5 (excluding fire, nether portal & end portal)
-	nonsolid = 0, 6, 27, 28, 30, 31, 32, 37, 38, 39, 40, 50, 55, 59, 63, 64, 65, 66, 68, 69, 70, 71, 72, 75, 76, 77, 78:0, 83, 85:4-7, 104, 105, 106, 111, 115, 131, 132, 141, 142, 143, 147, 148, 157
-
 on rightclick with compass:
 	player has permission "skript.teleport"
 	loop blocks above targeted block:

```

Comment:
```suggestion
	loop blocks above targeted block [where input isn't solid]:
```

Reply:
I'd rather use a simple if check rather than the where filter, it'd confuse new people.

Reply:
true